src-ui/lib/components/BooksTable.svelte

```diff
@@ -15,12 +15,13 @@
     hasRead: true,
   }
 
+
```

Comment:
Ok taking a first pass! I'm pretty new to PR reviews, so take every comment with a grain of salt 🙏 
Did prettier do this?